core/src/main/scala/org/apache/spark/h2o/utils/H2OContextUtils.scala

```diff
@@ -137,23 +137,10 @@ private[spark] trait H2OContextUtils extends Logging {
     }
   }
 
-  /**
-    * @param destination directory where the logs will be downloaded
-    */
-  def downloadH2OLogs(destination: URI, logContainer: LogArchiveContainer): URI = {
-    H2O.downloadLogs(destination, logContainer)
-  }
-
-  def downloadH2OLogs(destination: URI, logContainer: String): URI = {
```

Comment:
Removing the extra methods to keep simple api without the dependency on LogArchiveContainer for the future